README.md

```diff
@@ -137,8 +19,9 @@ Demo app that uses [prop=images](https://www.mediawiki.org/wiki/API:Images) modu
 * [User Contributions](python/demos/UserContributions)
 A sample app that uses MediaWiki Action [API:Usercontribs](https://www.mediawiki.org/wiki/API:Usercontribs) allows you to see the latest top 50 edits made by a user. This app uses Flask Framework as backend.
 
-### Installation
```

Comment:
For now, let's keep `Installation` & `Contributing code samples` as it is. We will address that in a separate pull request.